content/docs/connect/connection-latency.md

```diff
@@ -42,9 +42,9 @@ Given the potential impact on application responsiveness, it's important to have
 
 ### Adjust your Auto-suspend (scale to zero) configuration
 
-The [Neon Pro plan](/docs/introduction/pro-plan) allows you to configure the period before the system scales down to zero, providing you with control over the balance between performance and cost. The configuration setting is called **Auto-suspend delay**, and it is set to 300 seconds (5 minutes) by default. You can disable Auto-suspend entirely or increase the setting up to a maximum of 7 days. This strategy can eliminate or reduce compute startup times, but also increases compute usage. For configuration instructions, see [Edit a compute endpoint](/docs/manage/endpoints#edit-a-compute-endpoint).
+The [Neon Pro plan](/docs/introduction/pro-plan) allows you to configure the period before the system scales down to zero, providing you with control over the balance between performance and cost. The **Suspend compute after a period of inactivity** setting is set to 5 minutes by default. You can disable Auto-suspend entirely or set a custom period up to a maximum of 7 days. This strategy can eliminate or reduce compute startup times, but also increases compute usage. For configuration instructions, see [Edit a compute endpoint](/docs/manage/endpoints#edit-a-compute-endpoint).
```

Comment:
For me, the "inactivity" part of the equation doesn't seem clear enough. Consider:

"With the Neon Pro plan, you can configure the length of time that the system remains in an inactive state before Neon scales your compute down to zero. This lets you set the balance between performance (never scaling down) and cost (scaling to zero at reasonable intervals)."

Also consider adding some info in this sentence to make the pronoun reference clearer:

"Limiting or disabling autosuspend can eliminate or reduce startup times, but it also increases compute usage."

Reply:
Fixed. Thank you.